### PR TITLE
Deid 2564 deid examples remove accuracy high

### PR DESCRIPTION
**Pull request by AmirPAI**
https://privateai.atlassian.net/browse/DEID-2564

### What has changed?
1. Removed accuracy="high" from examples.
2. Fixed concurrency example which had errors.

### Note
All the *Python* examples changed have been tested by runs.

### CODE REVIEW

**Review by bryanbellsmith — APPROVED** [review]
One minor comment, but otherwise this looks like a good change to me!

File: python/examples/concurrency.py [hunk 1]
```diff
@@ -34,10 +34,9 @@ def make_request(request: request_objects.process_text_obj) -> Dict[str, str]:
     response = client.process_text(request)
 
     # check if the request was successful
-    response.raise_for_status()
-
-    # return the body of the response
-    return response.json()
+    if response.ok:
```
[the remark this answers is not in the document]
Comment (AmirPAI) [reply python/examples/concurrency.py:37]: I see but `response` here is a TextResponse object returned by our `client.process_text`. 
It doesn't have the `raise_for_status()` or `.json()` methods which are from the typical HTTP response object.

[the remark this answers is not in the document]
Comment (bryanbellsmith) [reply python/examples/concurrency.py:38]: Ah, I see, the thin client is only passing the dictionary. This is a good alternative then, thank you!

[the remark this answers is not in the document]
Comment (AmirPAI) [reply python/examples/concurrency.py:38]: Thanks!

**Pull request merged**